pages/event/[eventSlug]/ticket/index.vue

```
@@ -215,6 +225,9 @@ function goBack() {
 
 const SIGNOS_FEEDBACK_DELAY = 3000; // 3 seconds
 
+const lastPreScreening = localStorage.getItem(`lastPreScreening_${eventSlug}`);
+const lastQuestionnairePreScreening = lastPreScreening ? JSON.parse(lastPreScreening) : undefined;
```

ESLint

[error] 'lastQuestionnairePreScreening' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)